pyproject.toml

```diff
@@ -305,3 +305,14 @@ enable = [
     "deprecated-pragma",
     "invalid-name"
 ]
+
+[tool.bandit]
+exclude_dirs = [
+    "test",
+    ".eggs"
+]
+skips = [
+    "B101", # assert_used
+    "B105", # hardcoded_password_string too noisy on defaults values empty string
+    "B106", # hardcoded_password_funcarg too noisy on defaults values empty string
```

Comment:
These two fire any variable that has `token` in the name. We have a few places where we set empty string as opposed to None in order not to propagate None checks all over the place. It is debatable whether this is good code, but quite involved to fix.